src/main/scala/edu/berkeley/ce/sparkrocks/SeedJointSelector.scala

```diff
@@ -74,7 +74,7 @@ object SeedJointSelector {
     } else if (jointSets.head(0).shape.isEmpty) {
       val (seedJoints, leftOverJoints) = searchJointSets(jointSets, inputVolume, numPartitions, jointSetSpan)
       val partitioning = seedJoints.foldLeft(Seq(inputVolume)) { (currentBlocks, joint) =>
-        currentBlocks.flatMap(_.cut(joint))
+        currentBlocks.flatMap(_.cut(joint, minSize = 0.0, maxAspectRatio = Double.PositiveInfinity))
```

Comment:
Now that I think about it, why did we decide we couldn't give `minSize` and `maxAspectRatio` default values?

It seems like 0 and `Double.PositiveInfinity` are such a common use case, that it would still be valuable to have them as defaults, and we can just specify them explicitly in `RockSlicer`.